development/stream_interface/workflows_demo.py

```diff
@@ -1,36 +1,37 @@
 import os
 from threading import Thread
 
+import cv2
+import supervision as sv
+
 from inference import InferencePipeline
 from inference.core.interfaces.camera.entities import VideoFrame
-from inference.core.interfaces.stream.sinks import render_boxes
 from inference.core.interfaces.stream.watchdog import PipelineWatchDog, BasePipelineWatchDog
 
 STOP = False
+ANNOTATOR = sv.BoundingBoxAnnotator()
 
 
 def main() -> None:
     global STOP
     watchdog = BasePipelineWatchDog()
     workflow_specification = {
-        "specification": {
-            "version": "1.0",
-            "inputs": [
-                {"type": "WorkflowImage", "name": "image"},
-            ],
-            "steps": [
-                {
-                    "type": "ObjectDetectionModel",
-                    "name": "step_1",
-                    "image": "$inputs.image",
-                    "model_id": "yolov8n-640",
-                    "confidence": 0.5,
-                }
-            ],
-            "outputs": [
-                {"type": "JsonField", "name": "predictions", "selector": "$steps.step_1.*"},
-            ],
-        }
+        "version": "1.0",
```

Comment:
noting interface change, do we need to update examples / code snippets anyhere?  I think in app code snippets all use named workflow so are ok.

Reply:
if code snippets provide examples how to deal with result we may need to change them